elasticsearch/examples/default/test/goss.yaml

```
@@ -15,7 +15,7 @@ http:
     status: 200
     timeout: 2000
     body:
-      - '"number" : "7.9.0"'
+      - '"number" : "7.9.1"'
```

```suggestion
      - '"number" : "7.9.1-SNAPSHOT"'
```

related to https://github.com/elastic/helm-charts/blob/ce7f0873a15fb8d80ec2cd3c74eb8547da171f88/elasticsearch/examples/default/test/goss.yaml#L18